test/helpers.py

```diff
@@ -38,6 +38,10 @@ def compute_volume(points, cells):
                 ]).T
             mesh = voropy.mesh_tri.MeshTri(*prune_nodes(points, split_cells))
             vol += math.fsum(mesh.cell_volumes)
+    else:
+        assert 'line' in cells
+        segs = numpy.diff(points[cells['line']], axis=1).squeeze()
+        vol = numpy.sum(LA.norm(segs, axis=1))
```

Comment:
No need for scipy here; `einsum('i...,i...->...', segs, segs)` should be faster anyway.

Reply:
Made the change using einsum but it requires a sqrt for each segment then a final sum over all segments. But scipy is removed. See commit b1d3593a383d2c8799e36c14597d71a9a776ac10